ui/apps/platform/src/routePaths.js

```diff
@@ -131,7 +132,9 @@ export const basePathToLabelMap = {
     [apidocsPath]: 'API Reference',
     [productDocsPath]: 'Help Center',
     [clustersBasePath]: 'Clusters',
+    [policyManagementBasePath]: 'Policy Management',
     [policiesBasePath]: 'Policy Management',
+    [policyCategoriesPath]: 'Policy Categories',
```

Comment:
For your consideration, left nav uses these strings, but Access Control takes responsibility for page titles:
* Access Control - Auth providers
* Access Control - Auth provider
* Access Control - Roles
* Access Control - Role
* Access Control - Permission sets
* Access Control - Permission set
* Access Control - Access scopes
* Access Control - Access scope

Here is an application of that pattern:
* Policy management - Policies
* Policy management - Policy
* Policy management - Policy categories

Reply:
i'll address in the next PR, thanks